android/sdl_android/src/androidTest/res/xml/supported_vehicle_type.xml

```diff
@@ -5,4 +5,8 @@
         model="Mustang"
         modelYear="2019"
         trim="GT" />
+
+    <vehicle-type
+        make="Ford"
+        trim="GT" />
```

Comment:
What is the reason for adding this?

Reply:
I think we should reproduce a case when a developer didn’t define ‘make’ and ‘model’ attributes together but want to use ‘trim’ attribute. This data allows us to see behavior of the deserialization mechanism in SdlAppInfoTests.java in this case.

Reply:
@YaroslavLutsenko in that case I recommend either use other values for make and trim (because first `vehicle-type` tag is redundant and covered by the second one) or keep just one `vehile-type` in this file

Reply:
@AKalinich-Luxoft It's a good point. Thx. Added (see 1dd00a8)

Reply:
@YaroslavLutsenko not sure if Nissan is in our consortium :) It's better to use Toyota or Subaru who are the members

Reply:
@AKalinich-Luxoft It's my bad :( I decided to change ‘trim’ attribute to ‘modelYear’ because I don’t know which models are supported :) (see ccdbb98)